CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ x.x.x Release notes (yyyy-MM-dd)
 
 ### Enhancements
 
-* None.
+* Fail with `RLMErrorAddressSpaceExhausted` if unable to acquire more mmap memory.
```

Comment:
Return a recoverable `RLMErrorAddressSpaceExhausted` error rather than crashing when failing to mmap in Realm initialization or write commit.